cube/src/main/java/zone/cogni/asquare/cube/convertor/ModelToJsonConversion.java

```diff
@@ -229,11 +229,10 @@ private void reportMissedSubjects(Context context, String root) {
   private void reportUnprocessedTriples(Context context, String root) {
     if (!log.isWarnEnabled()) return;
 
-    Model remainingModel = ModelFactory.createDefaultModel();
-    remainingModel.add(context.model);
-    remainingModel.remove(context.alreadyProcessedModel);
+    Model remainingModel = context.model.difference(context.alreadyProcessedModel);
+    remainingModel.setNsPrefixes(conversionProfile.getContext().getPrefixes());
```

Comment:
good idea but this can be a long list (currenly 15 long or so?)... and most of them would be unused. so either we drop this line... or you make sure you only log namespaces which are relevant to the model. 

in case you do the latter this might have a performance impact... so then I would also add an option in the configuration to enable or disable this. if disabled it would fall back to the default (i.e. not setting namespaces)

Reply:
I'll change it to not add the prefixes then :)